ansible/roles/image-service/templates/sql/image-service-export.sql

```diff
@@ -0,0 +1,75 @@
+--  Function used for full DB exports
+CREATE OR REPLACE FUNCTION export_images() RETURNS void AS $$
+    BEGIN
+        COPY  
+        (
+            select
+            data_resource_uid AS "dataResourceUid",
+            split_part(original_filename, '||', 2) AS "occurrenceID",
+            CONCAT( '{{image_service_base_url}}{{image_service_context_path}}/image/proxyImageThumbnailLarge?imageId=', image_identifier) AS identifier,
+            regexp_replace(creator, E'[\\n\\r]+', ' ', 'g' ) AS creator,
+            date_taken AS created,
+            regexp_replace(title, E'[\\n\\r]+', ' ', 'g' ) AS title,
+            mime_type AS format,
+            regexp_replace(license, E'[\\n\\r]+', ' ', 'g' ) AS license,
+            regexp_replace(rights, E'[\\n\\r]+', ' ', 'g' ) AS rights,
+            regexp_replace(rights_holder, E'[\\n\\r]+', ' ', 'g' ) AS "rightsHolder",
+            CONCAT('{{image_service_base_url}}{{image_service_context_path}}/image/', image_identifier) AS "references",
```

Comment:
This is somehow problematic when not using ansible (docker, debs, etc). Just for the record.